app-misc/anki/anki-2.1.15.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
If you can, add Python 3.9. Make sure you have tested with all 2 (or 3) as appropriate though.

3.9 may not be possible due to deps.

Reply:
Due to dependencies not being ready, 3.9 is not possible at this time.

Reply:
No worries. :)